src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0 50px;
+  align-items: center;
+  background-color: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav_link {
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: 500;
+  line-height: 60px;
+  height: 60px;
+  display: block;
+}
+
+.video {
+  margin-right: 0;
+  box-sizing: border-box;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  height: 4px;
+  width: 37px;
+  background-color: #00acdc;
+  margin: 0 auto;
+  border-radius: 8px;
+  position: absolute;
+  top: 56px;
+}
+
+:hover {
+  color: #00acdc;
+}
```

Comment:
It seems that the ':hover' pseudo-class is not tied to a specific element. It should be applied to an element like '.nav__link:hover' to specify which element should change color on hover.